src/foam/nanos/logger/StdoutLoggerDAO.js

```diff
@@ -30,8 +36,16 @@ foam.CLASS({
       LogMessage lm = (LogMessage) getDelegate().put_(x, obj);
       if ( getEnabled() &&
            lm != null ) {
-        System.out.println(lm.getCreated() + ","+lm.getThread()+","+lm.getSeverity()+","+lm.getMessage());
-      } 
+        // Only write WARN, ERROR to SYSLOG in production to reduce
```

Comment:
I think your indentation is off by one character.

Reply:
corrected.